broker/data-access-layer/eventmesh/src/ApiServerClient.js

```diff
@@ -639,12 +699,30 @@ class ApiServerClient {
     }
     // Currently most callers are calling this function with allNamespaces: true, only metering jobs are calling without NS and defaults to. Should not be used in any other context.
     const namespaceId = opts.namespaceId ? opts.namespaceId : _.get(config, 'sf_namespace', CONST.APISERVER.DEFAULT_NAMESPACE);
+
+    const crdJson = this.getCrdJson(opts.resourceGroup, opts.resourceType);
+    const group = crdJson ? crdJson.spec.group : opts.resourceGroup;
+    const version = this.getCrdVersion(crdJson);
+    const plural = crdJson ? crdJson.spec.names.plural : _.lowerCase(opts.resourceType);
+
+    const pretty = _.get(opts, 'query.pretty');
+    const _continue = _.get(opts, 'query.continue');
+    const fieldSelector = _.get(opts, 'query.fieldSelector');
+    const labelSelector = _.get(opts, 'query.labelSelector');
+    const limit = _.get(opts, 'query.limit');
+    const resourceVersion = _.get(opts, 'query.resourceVersion');
+    const timeoutSeconds = _.get(opts, 'query.timeoutSeconds');
+    const watch = _.get(opts, 'query.watch');
```

Comment:
usually, from caller methods, we do not set any of these, right ?

Reply:
`labelSelector` is used somewhere. None of the other fields are used